### ISSUE TITLE
Actilino Braille device doesn't work with NVDA when locks the device and on the UAC prompt as well.  

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14202]
### Issue description: ###
User should be able to access the Logon UI when user tries to lock and then unlocks the device and also user should be able to access the UAC dialog of Command prompt i.e. the braille pins should get updated when NVDA is on.
 
### Test Environment: ###
Environment: Any RS_PreRelease branch above 25140 or 25140 
Braille Device: APH Refreshabraille 18 

###  Pre Requisite: ###
1. Actilino Device Setup is found here : Actilino (helptech.de) 
Third-party screen reader should be installed in system. Eg: to configure JAWS/NVDA, follow instructions as below 
a. Obtain a copy of the HTML version of the NVDA user guide. You may require this if you are testing a braille display that does not automatically get recognized by NVDA. Refer to section 15 of the NVDA User Guide entitled “Supported Braille Displays”. 
b. Download and install NVDA 2021.2. When asked if you want NVDA to start on startup or after sign-in during set up answer no. 
c. Complete NVDA setup. 
d. Check the NVDA User Guide for your specific braille displays that you will be testing with and download the needed drivers if any are required. The following models will be detected automatically by default for NVDA: 
e. Handy Tech displays 
f. Baum/Humanware/APH/Orbit braille displays 
g. HumanWare Brailliant BI/B series 
h. HumanWare BrailleNote 
i. SuperBraille 
j. Optelec ALVA 6 series 
k. HIMS Braille Sense/Braille EDGE/Smart Beetle/Sync Braille Series 
l. Eurobraille Esys/Esytime/Iris displays 
m. Nattiq nBraille displays 
n. Complete any additional setup required for the braille display as noted in the NVDA user guide. 
o. Download any other drivers required for other braille displays that you have for your testing efforts. 
p. Start NVDA and plugin the braille display. t
q. Make sure that the braille display is fully functional with NVDA. Make sure that NVDA is set to start before and after login from the NVDA Preferences, Settings dialog (Insert or Caps lock + N brings up the NVDA menu). Check these two checkboxes: ,
r. a. Start NVDA after I sign-in 
   b. Use NVDA during sign-in 
s. Execute the button labeled “Use currently saved settings during sign-in and on secure screens”. This required admin permissions and a UAC prompt will appear. This elevates your settings to be available at the lock screen. 
t. Stop and restart NVDA and make sure that braille continues to function. 
2.  Make sure braille package should be installed from Narrator Settings Page 
3.  Make sure braille device should be plugged in. 
4.  Make sure braille display driver should be other than libusb. If not follow these steps: 
   a. Open settings and search "Use a braille display with Narrator" button and click it. 
   b. Go to setting "Braille display driver" and change the driver to braille display driver other than libusb

### Repro steps: ###
1. Lock your screen using Windows + L and make sure that third-party screen-reader (NVDA) appears along with speech and braille. Login to the machine. 
2. Open a command line prompt with administrative permissions and make sure that when the UAC prompt appears that third-party screen-reader (NVDA) shows this information in braille. Third-party screen-reader (NVDA) will restart when the UAC window appears. 
3. Exit the command line and reboot the computer. 
4. After third-party screen-reader (NVDA) starts up make sure that braille appears at the log in window and after log in. 

### Actual result: ###
Actilino Device doesn't work with NVDA when user manually locks the screen and then tries to navigate in the Login screen and if we restart the machine then NVDA works on Login UI, this only reproes when user locks the screen. 

Actilino doesn't work ON UAC prompt of the CMD. 

### Expected result: ### 

Braille device should be functional all the time. 

### Attachments: ###
https://microsoft-my.sharepoint-df.com/:v:/p/v-shire/EZoUpVQezoBBiiVzFigdgFUB-K3Ymt9K6LsVgpNLEFF3LA?e=yv1zse

**Comment by seanbudd** [comment]
Closing as duplicate of #2315
